buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -4,14 +4,14 @@
 
 object Deps {
 
-    private const val kotlinVersion = "1.4.0"
+    private const val kotlinVersion = "1.4.31"
     private const val detektVersion = "1.7.4"
```

Comment:
update detekt to 1.15.0 for fix of `Could not find org.jetbrains.kotlinx:kotlinx-html-jvm`